### ISSUE TITLE
wiring botkit with testmybot and jasmine issue

### ISSUE THREAD

**Issue by nkkize** [codeforequity-at/testmybot#53]
**Issue:** Running `node jasmine.js` inside `samples/botkit/jasmine` gives below error:
**Date:** Feb 12, 2018
```
Started
DEPRECATION: done callback received an Error object. Jasmine 3.0 will treat this as a failure
FDEPRECATION: done callback received an Error object. Jasmine 3.0 will treat this as a failure
F

Failures:
1) TestMyBot Test Suite for testmybot-sample-botkit-jasmine Call Me Captain
  Message:
    Failed: Capability property STARTCMD not set
  Stack:
    Error: Capability property STARTCMD not set
        at DockerContainer._AssertCapabilityExists (/home/cpu409/Documents/workspace/westfield/testmybot/samples/botkit/jasmine/node_modules/botium-core/src/containers/BaseContainer.js:139:13)
        at Validate.Validate.then (/home/cpu409/Documents/workspace/westfield/testmybot/samples/botkit/jasmine/node_modules/botium-core/src/containers/DockerContainer.js:29:12)
  Message:
    Failed: container not available
  Stack:
    Error: container not available
        at TestMyBot.beforeEach (/home/cpu409/Documents/workspace/westfield/testmybot/samples/botkit/jasmine/node_modules/testmybot/src/testmybot.js:57:29)
        at UserContext.beforeEach (/home/cpu409/Documents/workspace/westfield/testmybot/samples/botkit/jasmine/node_modules/testmybot/src/helpers/jasmine.js:38:11)

2) TestMyBot Test Suite for testmybot-sample-botkit-jasmine Structured
  Message:
    Failed: Capability property STARTCMD not set
  Stack:
    Error: Capability property STARTCMD not set
        at DockerContainer._AssertCapabilityExists (/home/cpu409/Documents/workspace/westfield/testmybot/samples/botkit/jasmine/node_modules/botium-core/src/containers/BaseContainer.js:139:13)
        at Validate.Validate.then (/home/cpu409/Documents/workspace/westfield/testmybot/samples/botkit/jasmine/node_modules/botium-core/src/containers/DockerContainer.js:29:12)
  Message:
    Failed: container not available
  Stack:
    Error: container not available
        at TestMyBot.beforeEach (/home/cpu409/Documents/workspace/westfield/testmybot/samples/botkit/jasmine/node_modules/testmybot/src/testmybot.js:57:29)
        at UserContext.beforeEach (/home/cpu409/Documents/workspace/westfield/testmybot/samples/botkit/jasmine/node_modules/testmybot/src/helpers/jasmine.js:38:11)

2 specs, 2 failures
Finished in 1.362 seconds
```
As per [Readme.md](https://github.com/codeforequity-at/testmybot/blob/master/samples/botkit/jasmine/Readme.md), it should work without needing a Docker container.

**Comment by codeforequity-at** [comment]
Confirmed. This is a bug.

**Comment by nkkize** [comment]
any update on this? when can we expect a working solution?

**Comment by codeforequity-at** [comment]
You can expect a working solution with the next minor release (within the next 12 hours).

**Comment by codeforequity-at** [comment]
fixed with latest release 1.2.3
